package.json

```
@@ -24,7 +24,8 @@
     "serve": "cross-env BABEL_ENV=docs webpack-dashboard -- webpack-dev-server --content-base docs/ --open"
   },
   "files": [
-    "lib/"
+    "lib/",
+    "src/"
```

Not need, see [example](https://github.com/inside-demo/webpack-transpile-dependencies-demo)
```suggestion
    "src/"
```